common/src/main/java/com/distkv/common/exception/SetItemNotFoundException.java

```diff
@@ -0,0 +1,27 @@
+package com.distkv.common.exception;
+
+public class SetItemNotFoundException extends DistkvException {
+
+  protected String errorCode = ErrorCodeEnum
+      .SetItemNotFoundErrorCode
+      .getErrorCode();
+
+  protected String key;
+
+  public SetItemNotFoundException(String key) {
+    super(String.format("The item is not found in the sets %s",  key));
+  }
+
+  public SetItemNotFoundException(String key, String typeCode) {
```

Comment:
This constructor seems unused.

Reply:
Used in **com/distkv/client/CheckStatusUtil.java**